api/Node.json

```
@@ -383,10 +383,10 @@
               "version_added": "≤12.1"
             },
             "safari": {
-              "version_added": "7"
+              "version_added": "≤4"
```

```suggestion
              "version_added": "1"
```

Source: https://github.com/WebKit/WebKit/blob/b923c2f4eb7804aae65d67b4c3061d6435acb10d/WebCore/khtml/ecma/kjs_dom.cpp#L168-L169